src/Method/Handler/HandlerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SilverStripe\MFA\Method\Handler;
+
+use SilverStripe\MFA\Store\StoreInterface;
+
+/**
+ * Interface HandlerInterface
+ *
+ * @package SilverStripe\MFA\Method\Handler
```

Comment:
This (terribly autogenerated PHPStorm) doc block doesn't need to exist, it doesn't provide any value =)